src/appengine/handlers/cron/cleanup.py

```diff
@@ -630,39 +646,52 @@ def notify_closed_issue_if_testcase_is_open(testcase, issue):
     return
 
   # Check if there is ignore label on issue already. If yes, bail out.
-  if issue_tracker_utils.was_label_added(issue, data_types.ISSUE_IGNORE_LABEL):
+  if issue_tracker_utils.was_label_added(issue, policy.label('ignore')):
```

Comment:
will this work if policy.label('ignore') was None, same for similar cases in this file ?

Reply:
Yep. was_label_added explicitly checks this.